index.html

```diff
@@ -297,37 +297,36 @@ <h1 class="animated flip delay-2s" style="color: #fefefe;">Events</h1>
           <div class="parent event-cards">
             <div class="div1 event-card" id="myBtn" style="cursor: pointer;">
               <!-- <a href="pdf/vercera-pdf.pdf" target="_blank"> -->
-                <h3 style="color: #fefefe;">Vercera 
-                  
-                Code-A-Thon&nbsp;<span style="color: aqua; font-size: 17px; text-decoration: underline;">(Read More)</span>
-
-                </h3>
+                
+                <h3 style="color: #fefefe;">Vercera Code-Thon&nbsp;</h3>
```

Comment:
@aaradhya01 
if we don't add "read more" on event card. people might not click on the card to read more about the event. there should be some direction button to the event details.

Reply:
same goes with prizes

Reply:
mergint it for now. we can take this in next PR